src/Model/GenreManager.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Model;
+
+class GenreManager extends AbstractManager
+{
+    /**
+     *      La constante TABLE est ici inutile?
+     */
+    const TABLE = 'genre';
+
```

Comment:
Elle n'est pas inutile, il faut que tu la réutilises dans tes méthodes en dessous à la place d'écrire genre en dur je pense.
Je te mets l'info de remplacement là où il faut dans ton code dans un autre commentaire.

Reply:
corrected. surprisingly, despite PHPStorm showing smthg wrong, it works perfectly.